vlog4j-examples/src/main/java/org/semanticweb/vlog4j/examples/core/ConfigureReasonerLogging.java

```diff
@@ -69,11 +69,11 @@ public static void main(final String[] args) throws IOException, ParsingExceptio
 		try (final Reasoner reasoner = Reasoner.getInstance()) {
 			final KnowledgeBase kb = reasoner.getKnowledgeBase();
 			/* exists z. B(?y, !z) :- A(?x, ?y) . */
-			kb.addStatements(RuleParser.parseRule("B(?Y, !Z) :- A(?X, ?Y) ."));
```

Comment:
this is parserule function, not parse fact

Reply:
True. While the goal of the branch was to remove the parseFact method, I also updated how we are parsing rules into a KB.

Reply:
did you remove parserule ?

Reply:
The method `parseRule` is still part of the class `RuleParser`. Here I updated how we are adding some rules into a `KnowledgeBase`.

Users are still able to use `RuleParser.parseRule` in order to get an instance of `Rule`